packages/alea-frontend/components/HomeworkPerformanceTable.tsx

```diff
@@ -0,0 +1,130 @@
+import {
+  Box,
+  Paper,
+  Table,
+  TableBody,
+  TableCell,
+  TableContainer,
+  TableHead,
+  TableRow,
+  Typography,
+} from '@mui/material';
+
+import { PRIMARY_COL } from '@stex-react/utils';
+
+import Link from 'next/link';
+
+import { getLocaleObject } from '../lang/utils';
+import { useRouter } from 'next/router';
+
+function HomeworkPerformanceTable({ courseId }: { courseId: string }) {
+  const { homeworkPerformanceTable: t, homework: tHW } = getLocaleObject(useRouter());
+
+  const homeworkData = [
```

Comment:
constants in upper snake case.
homeworkData --> HOMEWORK_DATA

Reply:
Resolved.